### ISSUE TITLE
Add a warning threshhold per activity

### ISSUE THREAD

**Issue by helmo** [projecthamster/hamster#314]
Feature request ... wild idea ... just wondering.

In certain projects it can be useful to get a warning when the recorded amount of time gets over a certain amount.

My personal usecase would be to set a soft limit per activity, warning before starting a new activity and when the threshold is reached.

Implementing for every usecase might lead to feature creep ... a few possible variations might be: 
* threshold per day/week/month/year
* threshold per activity or category, or maybe even tag?
* softly warn using a popup notification uppon reaching the threshold
* hard warn, and stop the time tracking uppon reaching the threshold
* softly warn when starting a new activity 
* hard block from starting a new activity

**Comment by elbenfreund** [comment]
If you don't mind sharing, what is your usecase for this warning?
Is it about not spending too much time "doing X"?

**Comment by helmo** [comment]
Exactly. I have a few clients where I have a 'budget' of X hours a month to do stuff for them. When I spent more time I discuss with them upfront.
